apps/core/components/ItineraryCard/itineraryDetail/ItineraryDetail.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import * as React from 'react';
+import { View, Platform } from 'react-native';
+import { Button, StyleSheet, Text } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import ItineraryDetailWrapper from './ItineraryDetailWrapper';
+
+// todo
+type Props = {|
+  +localizedPrice: string,
+  +onClose: () => void,
+|};
+
+export default class ItineraryDetail extends React.Component<Props> {
+  handleBookPress = () => {
+    // @TODO redirect to booking screen
+  };
+
+  render() {
+    const { localizedPrice, onClose } = this.props;
+
+    return (
+      <ItineraryDetailWrapper onClose={onClose}>
+        <View style={styles.container}>
+          <View style={styles.content}>
+            <Text>TODO: Detail data</Text>
+          </View>
+          <View style={styles.footer}>
+            {Platform.OS !== 'web' && (
+              <Button type="secondary" label="CLOSE" onPress={onClose} />
+            )}
+            <View style={styles.bookButtonWrapper}>
+              <Button
+                label={`BOOK FOR ${localizedPrice}`}
+                onPress={this.handleBookPress}
+              />
+            </View>
+          </View>
+        </View>
+      </ItineraryDetailWrapper>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'flex-end',
+  },
+  content: {
+    flex: 1,
+    padding: parseInt(defaultTokens.spaceMedium, 10),
+  },
+  footer: {
+    flexDirection: 'row',
+    justifyContent: 'flex-end',
+    padding: parseInt(defaultTokens.spaceMedium, 10),
+  },
+  bookButtonWrapper: {
+    flex: 1,
+    marginStart: parseInt(defaultTokens.spaceXSmall, 10),
+    web: {
+      flex: -1,
```

Comment:
What does it do? 😮

Reply:
I am quite sure it is invalid value

Reply:
This one simply resets wrapper to original behavior. 
https://facebook.github.io/react-native/docs/layout-props#flex

If I use `0` something like this will happen.
<img width="243" alt="screenshot 2019-02-21 at 10 04 06" src="https://user-images.githubusercontent.com/2660330/53156820-78e86580-35c0-11e9-9329-6b8f2abef69c.png">

Reply:
What about `null`?

Reply:
Right, it will also work and it is definitely less confusing. Thanks, updated.